[es] Translation of intro.md based on 49985f9432

franktoffel opened the pull request:
https://github.com/franktoffel/swcarpentry-es/blob/49985f9432ac3d272a507dc3ec315c242872f265/intro.md

franktoffel committed:
[es] Translation of intro.md based on 49985f9432

https://github.com/franktoffel/swcarpentry-es/blob/49985f9432ac3d272a507dc3ec315c242872f265/intro.md

astrojuanlu commented on translations/es/intro.md:
```diff
@@ -4,86 +4,62 @@ root: .
+El mayor dolor de cabeza de los investigadores proviene de no saber cómo desarrollar software de forma sistemática, cómo determinar si sus programas están funcionando correctamente, cómo compartir su trabajo con otros (excepto por archivos adjuntos vía email) o cómo monitorizar los avances que han hecho. Esta situación persiste por cuatro razones:
+
+*   *Sin espacio, sin tiempo.*
+    La agenda de todo el mundo está llena, simplemente no hay espacio para añadir más sobre computación científica sin tener que decir que no a alguna otra cosa.
```
Parece que originalmente se cortaban las líneas en unos 80 caracteres pero en la traducción has dejado una por frase. No sé si era tu intención, en caso contrario hay editores que ayudan a cortar las líneas (nano lo hace automáticamente, y con Kate tienes Tools -> Apply Word Wrap.

dpshelio replied:
Estoy de acuerdo con @Juanlu001 - y deberíamos añadirlo a las instrucciones para futuros traductores. Yo sugeriría empezar cada frase con una nueva línea, y no más largas de 80 caracteres.  Eso hará más fácil en el futuro comparar que se ha cambiado entre una versión y otra.

franktoffel replied:
Ok, como hice la traducción con el editor online no estaba seguro de si se tendrían problemas a la hora de pasarlo a HTML. Lo pongo ahora en el README

astrojuanlu commented on translations/es/intro.md:
```diff
@@ -106,17 +82,21 @@ Software Carpentry has been made possible by the generous support of:
-Special thanks go to Brent Gorda,
-who helped build and teach the first version of this course.
+Gracias especialmente a Brent Gorda,
+quien ayudó a construir y enseñear la primera versión de este curso.
```
*enseñar

dpshelio replied:
que tal **impartir** ?? enseñar me suena un poco raro...

franktoffel replied:
impartir a mi también me suena mejor

astrojuanlu commented on translations/es/intro.md:
```diff
@@ -106,17 +82,21 @@ Software Carpentry has been made possible by the generous support of:
+
+### Traducción
+
+Las lecciones de este curso han sido traducidas del Inglés por [voluntarios](https://github.com/franktoffel/swcarpentry-es/tree/master/translations/es#colaboradores) y tienen como objetivo facilitar (que no sustituir) la versión original de las mismas. 
```
*inglés

dpshelio replied:
La voz pasiva hace a veces que el texto parezca muy pesado.  Yo diría:
...curso **se han traducido** del inglés ...
Pero esto es una preferencia personal mía, así que como tu prefieras :)

franktoffel replied:
Estoy de acuerdo, mejor evitar la voz pasiva

astrojuanlu commented:
Como estaba un poco harto de estudiar he echado un vistazo a lo que has escrito... :)

dpshelio commented on translations/es/intro.md:
```diff
@@ -4,86 +4,62 @@ root: .
+Este es el sueño:
+
+> Los ordenadores han revolucionado la investigación, y esta revolución
+> está únicamente empezando. Cada día, científicos e ingenieros de todo
```
está únicamente empezando -> sólo acaba de empezar ??

franktoffel replied:
Es más natural como dices, tuve mis dudas en ese momento y lo cambié

dpshelio commented on translations/es/intro.md:
```diff
@@ -4,86 +4,62 @@ root: .
+> está únicamente empezando. Cada día, científicos e ingenieros de todo
+> el mundo los emplean para estudiar cosas que son demasiado grandes,
+> demasiado pequeñas, demasiado rápidas, demasiado lentas, demasiado caras,
+> demasiado peligrosas, o simplemente demasiado difíciles para estudiar de otra manera.
```
para estudiar de otra manera -> ...estudiarlas... ??

franktoffel replied:
Ok

dpshelio commented on translations/es/intro.md:
```diff
@@ -4,86 +4,62 @@ root: .
+    La agenda de todo el mundo está llena, simplemente no hay espacio para añadir más sobre computación científica sin tener que decir que no a alguna otra cosa.
+
+*   *Sin estándares.*
+    Los reviewers y agencias que subvencionan proyectos no comprueban si el software es correcto, preguntan cuánto llevo escribirlo o lo valoran según antigüedad, con lo que no hay incentivos para que los científicos lo hagan mejor.
```
llevo -> llevó

No estoy seguro del sentido de "antigüedad" en esta frase.  Del original pensaría que se refiere al autor, a cuanto lleva una persona en su puesto en la universidad o instituto.  En la traducción creo que se refiere a cuánto antiguo es el software... Y sinceramente no estoy del todo convencido cual es la versión correcta.
Si es la que entiendo yo del inglés recomendaría cambiarla por:

>  ...o lo valoran según antigüedad del científico/de la persona,

o tal vez usar "experiencia" (aunque se puede malinterpretar en este contexto...), no sé ¿qué se usa en las universidades españolas?

Si se cambia, creo que habría que cambiar el final de la frase para que sonase mejor

> lo que conlleva una falta de incentivo para que el software se mejore.

franktoffel replied:
Pues ahora mismo tampoco tengo claro como se dice, lo preguntaré por si acaso. Experiencia lo descarté porque puede, como has dicho, dar a entender justo lo contrario. 

Antigüedad lo veo bien.

dpshelio commented on translations/es/intro.md:
```diff
@@ -4,86 +4,62 @@ root: .
+    Los reviewers y agencias que subvencionan proyectos no comprueban si el software es correcto, preguntan cuánto llevo escribirlo o lo valoran según antigüedad, con lo que no hay incentivos para que los científicos lo hagan mejor.
+
+*   *El ciego guiando a los ciegos.*
+    Los investigadores senior no pueden enseñar a la siguiente generación cómo hacer cosas que ellos no saben hacer por sí mismos.
```
¿se usa senior en español? No se me ocurre otra palabra, pero senior me choca.

franktoffel replied:
He estado preguntando y varias personas del entorno científico utilizan este término.

dpshelio replied:
perfecto entonces :)

dpshelio commented on translations/es/intro.md:
```diff
@@ -4,86 +4,62 @@ root: .
+*   *El culto al poder.*
+    La atención y los fondos van mayoritariamente hacia cosas de las que los políticos y los rectores de universidades puedan alardear en inauguraciones, más que a las habilidades básicas que casi todo el mundo emplea.
+
+Nuestra misión es enseñar a los investigadores habilidades básicas informáticas aplicables en el ámbito científico, herramientas y técnicas que ayudarán a hacer más cosas en menos tiempo y de forma menos dolorosa. Los instructores son voluntarios y han impartido más de un centenar de eventos a miles de científicos en los últimos dos años y medio. Todos los materiales de las lecciones son libremen ...
```
de forma menos dolorosa - con menor sufrimiento ??

En este párrafo has introducido texto que no proviene del original, no tengo nada en contra de ello, pero creo que no estaría de más añadir algo antes del párrafo que nos haga saber que es texto propio.  Por ejemplo, yo pondría algo como:
`[//]: # (Texto añadido por el traductor)`
O algo por el estilo.  Esa parece ser la manera [de añadir comentarios en markdown](http://stackoverflow.com/a/20885980/1087595) que no se muestran cuando se reproduce/convierte a html (u otro formato).

dpshelio replied:
> Todos los materiales de las lecciones son libremente reutilizables bajo la licencia Creative Commons, Sin embargo, 

Me suena un poco raro, yo diría:

Todo el material de las lecciones se pueden reutilizar libremente gracias a su licencia Creative Commons**.** Sin embargo,

También, creo que sería mejor poner esto de la traducción al final, donde tienes la sección de traducción.

franktoffel replied:
En realidad introduje eso sin querer al ciopiar y pegar del README, mejor quitarlo ya que más tarde se habla de la traducción.
Por otro lado, añado como hacer comentarios en Markdown al README.

dpshelio commented on translations/es/intro.md:
```diff
@@ -4,86 +4,62 @@ root: .
+
+Nuestra misión es enseñar a los investigadores habilidades básicas informáticas aplicables en el ámbito científico, herramientas y técnicas que ayudarán a hacer más cosas en menos tiempo y de forma menos dolorosa. Los instructores son voluntarios y han impartido más de un centenar de eventos a miles de científicos en los últimos dos años y medio. Todos los materiales de las lecciones son libremen ...
+
+Nuestra meta es mostrar a los científicos e ingenieros cómo hacer más en menos tiempo y de forma menos dolorsa. Nuestras lecciones han sido empleadas por más de cuatro mil principiantes en alrededor de cien talleres de dos días desde la primavera de 2010.
```
déjà vu? :) 
dolorsa - sufrimiento ?
han sido empleadas por más - se han impartido a más?

franktoffel replied:
Sí, quizás sea mejor sufrimiento

Impartido (ok)

dpshelio commented on translations/es/intro.md:
```diff
@@ -4,86 +4,62 @@ root: .
+
+*   Si alguna vez has sobreescrito el archivo erróneo, te mostraremos cómo emplear un controlador de versiones (*version control*).
+*   Si alguna vez has pasado horas escribiendo los mismos comandos una y otra vez, te mostraremos cómo automatizar estas tareas empleando scripts simples.
+*   Si alguna vez has pasado una tarde intentando descubrir lo que el programa que escribiste la semana pasada realmente hace, te mostraremos cómo dividir tu código en módulos que puedas leer, eliminar errores (*bugs*) y mejorarlo bloque por bloque.
```
Si alguna vez has pasado una tarde intentando descubrir que (es lo que) hace el programa que escribiste la semana pasada ??

eliminar vs corregir errores?? No sé...  Corregir creo que es más apropiado, porque uno corrige algo que ha hecho mal... eliminar no da la impresión de que lo hemos hecho mal nosotros :-)

franktoffel replied:
corregir mejor :+1:

dpshelio commented on translations/es/intro.md:
```diff
@@ -4,86 +4,62 @@ root: .
+
+### Sobre nosotros
+
+Software Carpentry es un proyecto *open source* (código abierto).
```
anteriormente has traducido el texto y puesto entre paréntesis el texto en cursiva y en inglés.  Cualquiera de las dos me vale, pero tendremos que ser consistentes a lo largo de todo la traducción (algo que añadir a las recomendaciones)?

franktoffel replied:
Creo que mejor poner el término en inglés en cursiva seguido de su traducción entre paréntesis la primera vez que aparezca en el texto. Las siguientes se pone el término en inglés sin cursiva directamente. Si lo ves bien así, lo añado al README

dpshelio commented on translations/es/intro.md:
```diff
@@ -4,86 +4,62 @@ root: .
+### Sobre nosotros
+
+Software Carpentry es un proyecto *open source* (código abierto).
+Nuestros instructores son voluntarios y todas las lecciones son libremente reutilizables bajo licencia [Creative Commons - Atribución](http://creativecommons.org/licenses/by/3.0/es/deed.es), por lo que puede reutilizar y modificar todo el material teniendo solo que citarnos como fuente original.
```
En el texto original dice "disponibles" no "reutilizables" ... entiendo que en este caso no cambia mucho puesto que CC permite la reutilización... pero no sólo eso, también permite la modificación en incluso su uso comercial.

dpshelio commented on translations/es/intro.md:
```diff
@@ -4,86 +4,62 @@ root: .
+
+Para descubrir más, por favor visita [Software-Carpentry.org](http://software-carpentry.org) o lee 
+[estos](http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1001745)
 [papers](http://arxiv.org/abs/1307.5448)
```
papers - artículos

dpshelio commented:
@franktoffel espero no haberte asustado llenando tu correo con correcciones/comentarios :)
Me ha gustado leer la traducción! :)

Un par de cosas más:
En github puedes enlacer con el "issue" sólo poniendo algo como:

>   Esta PR resolvería #1 

Así, si vas a la página del issue te dice que se ha hecho referencia en esta PR.

También, creo que será mejor añadir los nombres de las personas y las instrucciones en una página de la wiki, así no vamos añadiendo commits al repositorio cada vez que uno se ofrezca voluntario - cosa que podremos añadir justo antes de hacer la PR al repositorio original.  También la wiki es editable por cualquier persona, así te alivia el tener que escribir un nombre cada vez.

franktoffel commented:
Todo lo contrario @dpshelio ! Con este sistema peer review va a salir una traducción espectacular!

Anoto lo que dices, aunque... si hay más voluntarios el menor de los problemas es añadirlos al README ;)

Esta tarde hago todos los cambios que hemos planteado

PD: Gracias @Juanlu001 por revisarlo tb! Ánimo con los exámenes

dpshelio commented:
@franktoffel entiendo que no sea un problema añadirlos al readme, como las correcciones del texto, el problema es que puede traer conflictos si los que colaboran bifurcan una versión anterior.  Yo optaría por tocar el repositorio directamente lo menor posible... mi experiencia me intenta apartar de los conflictos lo más que pueda :)

Por cierto, 
expectacular -> e_s_pectacular 
(uy, ya estoy empezando a ser un poco pedante) :-P

Ah, [wiki](https://github.com/franktoffel/swcarpentry-es/wiki) hecha - todavía necesita un poco de amor...

franktoffel commented:
Tienes razón (debería tener un acceso directo para pegar esta frase :P). Lo cierto es que una de las razones por las que propuse traducir estas lecciones era para aprender y poner en práctica todo lo que estamos comentando. Así que, sin piedad! :)

Completaré la guía de estilo con más detalles en la wiki en cuanto tenga tiempo

franktoffel committed:
[es] Changes in intro.md based on 49985f9432 PR

https://github.com/franktoffel/swcarpentry-es/blob/49985f9432ac3d272a507dc3ec315c242872f265/intro.md
